s2s/time.py

```diff
@@ -170,6 +174,50 @@ def map_years(
 
         return index
 
+    def map_to_data(self, input_data: Union[pd.Series, pd.DataFrame, xr.Dataset, xr.DataArray],
+        flat: bool = False
+        ) -> pd.DataFrame:
+        """Map the calendar to input data period.
+        
+        Get datetime range from input data and generate corresponding interval index. This method
+        guarentess that the generated interval (calendar) indices would be covered by the input
```

Comment:
missed this typo last time, sorry! (`guarantees`)

Reply:
Sorry @BSchilperoort . My bad, I will correct it. Thanks for spotting!